cap-2.x/gke/main.tf

```diff
@@ -41,25 +42,46 @@ module "helper-charts" {
   ]
 }
 
-module "cap-charts" {
-  source = "./common/cap-charts"
+module "kubecf-charts" {
+  source = "./common/kubecf-charts"
 
   providers = {
     helm = helm.helm-cap
   }
 
   cap_domain = var.cap_domain
-  cluster_url = module.gke-cluster.cluster_url
 
   # One variable is initially applied to all security contexts,
   # override to create distinct passwords for each context
   cf_admin_password       = var.admin_password
   uaa_admin_client_secret = var.admin_password
-  stratos_admin_password  = var.admin_password
-  metrics_admin_password  = var.admin_password
+  eirini_enabled          = var.eirini_enabled == "true" ? "true" : "false"
```

Comment:
Since the var has a default, do we need to do the ternary setting?

Reply:
It's because if the operator passes in a value such as eirini_enabled=no or anything other than false helm install would fail.